ice/agents/openai.py

```diff
@@ -19,7 +19,7 @@ class OpenAIAgent(Agent):
     def __init__(
         self,
         model: str = "text-davinci-002",
-        temperature: float = 0.0,
```

Comment:
this should definitely not be merged.
it'd maybe be easier if this was a parameter of `call` rather than `__init__` - but perhaps there's some design philosophy part i'm missing